### ISSUE TITLE
UUID error

### ISSUE THREAD

**Issue by ItzMeWilliam** [Netriza/mrsdb#2]
If installing from pip, also installs PyPi `uuid` which overrides the built-in `uuid`, causing mrsdb not to work. 
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/mrsdb2/mrsdb.py", line 16, in <module>
    import bson
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/bson/__init__.py", line 22, in <module>
    from .codec import *
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/bson/codec.py", line 13, in <module>
    from uuid import UUID
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/uuid.py", line 138
    if not 0 <= time_low < 1<<32L:
                                ^
SyntaxError: invalid syntax
```
## Fix for now if anyone has this issue:
```
pip -m uninstall uuid
```

**Comment by ItzMeWilliam** [comment]
`uuid` in setup.py install requirements will be removed next commit.

**Comment by AyzansAlt** [comment]
teach me python

**Comment by ItzMeWilliam** [comment]
> teach me python

get on discord. and stop using issues for off-topic stuff.

**Comment by ItzMeWilliam** [comment]
Fixed in 2.2.2.
